src/app/features/product-page/containers/product-page/product-page.component.html

```diff
@@ -1,18 +1,56 @@
 <div class="product-card">
   <ile-images-slider [images]="product?.pictureLinks" [imageHeight]="200" [imageWidth]="'75rem'"></ile-images-slider>
   <div class="product-header">
-    <span class="product-name">{{ product?.name }}</span>
+    <div style="display: flex; align-items: center;">
+      <span class="product-name">{{ product?.name }}</span>
+      <span class="product-rating" *ngIf="product?.rating">{{product?.rating | number: '1.0-2'}}</span>
+    </div>
     <span class="product-price">${{ product?.requestedPrice }}</span>
   </div>
   <!--  TODO: Add router link to the user profile-->
   <div class="owner-label">Lend from <a>{{product?.owner.firstName + ' ' + product?.owner.lastName}}</a></div>
   <p class="product-description text-ellipsis">{{ product?.description || 'The owner didn\'t add any description' }}</p>
   <ile-button [text]="'Lend now'" class="lend-button" matTooltip="{{!loggedInUser ? 'Login to continue' : ''}}"
               [matTooltipPosition]="'above'" [disabled]="!loggedInUser"></ile-button>
-  <ile-button [text]="'Add to Wishlist'" *ngIf="!isProductInWishlist"
+  <ile-button [text]="'Add to Wishlist'" *ngIf="!isProductInWishlist" class="wishlist-button"
               matTooltip="{{!loggedInUser ? 'Login to continue' : isLoggedInUserProduct ? 'Can\'t add your own product to wishlist' : ''}}"
               [matTooltipPosition]="'above'" [disabled]="!loggedInUser || isLoggedInUserProduct"
               (clicked)="addProductToWishlist()"></ile-button>
-  <ile-button [text]="'Remove from Wishlist'" *ngIf="isProductInWishlist"
+  <ile-button [text]="'Remove from Wishlist'" *ngIf="isProductInWishlist" class="wishlist-button"
               (clicked)="removeProductFromWishlist()"></ile-button>
+  <div class="ratings" *ngIf="product?.owner.id !== loggedInUser?.id">
+    <fieldset class="rating" [ngStyle]="{'pointer-events': isRated ? 'none' : 'auto'}">
+      <input type="radio" id="star5" name="rating" value="5" (input)="updateProductRating(5)" /><label class = "full" for="star5"></label>
```

Comment:
Can't that be a for loop?
It seems it can make it look less scarry

Reply:
To be honest I took it from codepen.. I improved it to two for loops and it's still working. I tried make it with one for loop and all the styles broken so I think we keep it like that

Reply:
Sounds about right